AuthPages/auth.css

```diff
@@ -0,0 +1,42 @@
+.auth-container {
+  width: 100%;
+  height: 90%;
+  background-color: var(--violet-color-light);
+}
+
+.auth-card {
+  width: 30rem;
+  height: 30rem;
+  padding: 2rem 3rem;
+  background-color: var(--white);
+  border: 1px solid var(--grey);
+  border-radius: 7px;
+}
+
+.auth-card.logout {
+  height: auto;
+}
+
+.auth-card button {
+  width: 100%;
+}
+
+.check-container {
+  display: flex;
+  gap: 7px;
+  font-size: 1.1rem;
+  align-items: center;
+}
+
+.check-container input {
+  width: 1.1rem;
+  height: 1.1rem;
+}
+
+.check-container input:hover {
```

Comment:
This can also be done as
.check-container input:hover,
.forgot-pswd:hover{
cusor:pointer
}
Just a suggestion to reduce line of code

Reply:
okay will change it!